auth.go

```diff
@@ -168,6 +192,17 @@ func (a *Auth) Op(key ssh.PublicKey, d time.Duration) {
 	} else {
 		a.ops.Set(authItem)
 	}
+
+	// nick registration
```

Comment:
That occurred to me the instant I copied and pasted between these two functions, naturally :)

wrt to `@` , I actually want to keep the `@host` suffix. I tend to generate keys on each account on each host, rather than copying them around, so using them to identify accounts on particular hosts, rather than just individual people, seems natural to me.